test/parallel/test-fs-truncate.js

```diff
@@ -224,13 +224,14 @@ function testFtruncate(cb) {
 }
 
 {
-  const file6 = path.resolve(tmp, 'truncate-file-6.txt');
-  fs.writeFileSync(file6, 'Hi');
-  const fd = fs.openSync(file6, 'r+');
-  process.on('beforeExit', () => fs.closeSync(fd));
-  fs.ftruncate(fd, -1, common.mustSucceed(() => {
-    assert(fs.readFileSync(file6).equals(Buffer.from('')));
-  }));
+  assert.throws(
+    () => fs.ftruncate(fd, -1),
+    {
+      code: 'ERR_INVALID_ARG_VALUE',
+      name: /(TypeError|RangeError)/,
+      message: 'The argument \'len\' is invalid. Received -1'
+    }
+  );
```

Comment:
@IgorHalfeld The `fd` you have used here is already closed by the time this is run but the test passes because `ERR_INVALID_ARG_VALUE` is thrown earlier:
https://github.com/nodejs/node/blob/7efada695f9cb7515a78565213f25d8077c206ff/test/parallel/test-fs-truncate.js#L71

Instead of replacing the previous test completely, consider replacing only this part with your test (you may take a look at the other tests for help):
https://github.com/nodejs/node/blob/7efada695f9cb7515a78565213f25d8077c206ff/test/parallel/test-fs-truncate.js#L231-L233

Also, `fs.ftruncate` needs to be called with a callback: https://nodejs.org/api/fs.html#fs_fs_ftruncate_fd_len_callback

Similarly, consider adding a corresponding test for `fs.ftruncateSync` too: https://nodejs.org/api/fs.html#fs_fs_ftruncatesync_fd_len